instrumentation/opentelemetry-instrumentation-mysql/tests/test_mysql_integration.py

```diff
@@ -117,3 +117,25 @@ def test_uninstrument_connection(self, mock_connect):
 
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
+
+    @mock.patch("mysql.connector.connect")
+    # pylint: disable=unused-argument
+    def test_sqlcommenter_enabled(self, event_mocked):
+        MySQLInstrumentor().instrument(enable_commenter=True)
+        cnx = mysql.connector.connect(database="test")
+        query = "SELECT * FROM test"
+        cursor = cnx.cursor()
+        cursor.execute(query)
+        kwargs = event_mocked.call_args[1]
+        self.assertEqual(kwargs["enable_commenter"], True)
+
+    @mock.patch("mysql.connector.connect")
+    # pylint: disable=unused-argument
+    def test_sqlcommenter_disabled(self, event_mocked):
```

Comment:
@rahuldimri these tests are [failing](https://github.com/open-telemetry/opentelemetry-python-contrib/actions/runs/5525752108/jobs/10079715535?pr=1834#step:6:6911), please take a look. This PR has enough approvals, we only need these tests to pass in order to merge it.

Reply:
Sure let me work on this.

Reply:
@ocelotl let me have a look !!

Reply:
@rahuldimri 

Are you still working on this? Still seems like there are test cases that are failing.

Reply:
Hi @rahuldimri , are you still working on this PR?